### ISSUE TITLE
[Feature Idea] indented text import with cell styling

### ISSUE THREAD

**Issue by Sash-at-Github** [aardappel/treesheets#351]
I really like the the indented text import feature how's it allows for creating quick set of notes that later can be imported into treesheets and create a nice structured mind map look. But what would be nice is to add to this import feature the ability to specify a text colour or cell colour in the indented text.

For instance:
Note parent _blue_
 Note child  _green_

Something in the spirit of https://www.text2mm.com/ What I'm note keen on with text2mm is branches are positioned in a clockwise circle which can make them difficult to read. Treesheets trumps this on presentation.

**Comment by aardappel** [comment]
So is this for a workflow where you'd keep editing it as text and only import to TS for visualization?

I'm afraid that sounds a bit very niche to me.. also because its hard to invent a good stylization syntax for text a la Markdown since in TS the stylization is per indented item..

**Comment by Sash-at-Github** [comment]
Yes absolutely.. I take your point though that it is more than likely to be very niche. TS has some very interesting use cases particularly because of it's ease of use and speed. But perhaps for the the aforementioned workflow I might have to resort to something like d2lang.com- ~ -
